src/main/java/pl/gov/coi/pomocua/ads/legalassistances/LegalAssistanceResource.java

```diff
@@ -0,0 +1,36 @@
+package pl.gov.coi.pomocua.ads.legalassistances;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+
+@RestController
+@RequestMapping(path = "/api/", produces = "application/json")
+public class LegalAssistanceResource {
+
+    private final LegalAssistanceRepository legalAssistanceRepository;
+
+    public LegalAssistanceResource(LegalAssistanceRepository legalAssistanceRepository){this.legalAssistanceRepository = legalAssistanceRepository;}
+
+    @GetMapping(value = "legal_assistances")
```

Comment:
What do you think about using kebab-case (i.e. `legal-assistances`) here?
I think that snake_case is rarely used in REST APIs paths.

Reply:
You are right. Changing :)